### PR TITLE
[examples] blue-pill ITM logging

### PR DESCRIPTION
**Pull request by strongly-typed**

### CODE REVIEW

File: examples/blue_pill_f103/itm/main.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+	LedGreen::set();
+
+	while (true)
+	{
+		static modm::PeriodicTimer tmr{500ms};
+
+		if (tmr.execute())
+		{
+			tmr.restart(500ms);
```
Comment (salkinium) [inline examples/blue_pill_f103/itm/main.cpp]: Not necessary for a periodic timer?

Comment (rleh) [reply examples/blue_pill_f103/itm/main.cpp]: Restarting the periodic timer with every `tmr.execute()` event will create al lot jitter. (This is not critical here, but from my experience example code is copied and extended very often...)

Comment (strongly-typed) [reply examples/blue_pill_f103/itm/main.cpp]: You are right, this was copied from the F303 example. But this example uses
```C++
tmr.restart(Button::read() ? 100ms : 500ms);
```
Fixed.

**Review by rleh — APPROVED** [review]

**Review by salkinium — APPROVED** [review]

**Comment by strongly-typed** [comment]
Does the CI hang again? The `*-compile-all` jobs did not report their status ...

LGTM!

**Comment by salkinium** [comment]
No, I just went to bed before merging.
The compile all is triggered manually now, but not necessary here.

**Comment by strongly-typed** [comment]
Thanks! I was just confused by the checks that were not triggered.

**Pull request merged**